framework/helpers/docker.py

```diff
@@ -0,0 +1,308 @@
+from datetime import datetime
+from datetime import timedelta
+from math import ceil
+from pathlib import Path
+from pathlib import PosixPath
+from queue import Queue
+from threading import Thread
```

Comment:
here and elsewhere:

> Use import statements for packages and modules only, not for individual types, classes, or functions.
> — https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Done

Reply:
I do not know how to comply with linters other then importing individual protobuf generated classes. I made a change.

Reply:
Here's how:
```diff
- from protos.grpc.testing.xdsconfig.control_pb2 import StopOnRequestRequest
- from protos.grpc.testing.xdsconfig.control_pb2 import UpsertResourcesRequest
+ from protos.grpc.testing.xdsconfig import control_pb2
```

Then just use `control_pb2.StopOnRequestRequest` and `control_pb2.UpsertResourcesRequest`.

Reply:
Done